grid/language/named-entity-recognition/profile.supr

```diff
@@ -0,0 +1,125 @@
+"""
+Named-Entity Recognition
+Identify and classify named entities such as people, organizations, locations, dates, and other categories of words in a given text.
+"""
+name = "language/named-entity-recognition"
+version = "1.0.0"
+
+"""
+Named-Entity Recognition
+Identify and classify named entities such as people, organizations, locations, dates, and other categories of words in a given text.
```

Comment:
dup to the profile description? omit one of those perhaps?

Reply:
I opted for duplication so that we render something at each level 🤷‍♂️